### ISSUE TITLE
Couldn't add user at Newly installed Senaite

### ISSUE THREAD

**Issue by noob-git** [senaite/senaite.core#1465]
## Description

I couldn't add user from Site Setup --> Users and Groups. I've also tried to enable user self registration from Site Setup --> Security, but resulting the same error. 

Version overview of my newly installed Plone and Senaite:
Plone 4.3.18 (4320)
CMF 2.2.10
Zope 2.13.28
Python 2.7.13 (default, Nov 1 2019, 04:14:19) [GCC 7.4.0]
PIL 5.1.0 (Pillow)

## Reproduction guide

1. Install and setup a new Senaite server
2. Login using Zope admin account and password
3. Go to Site Setup --> Users and Groups
4. Try to register new user with any privilege
6. Go to Site Setup --> Security

## Observed behaviour

Resulting in error:

Traceback (innermost last):
  Module ZPublisher.Publish, line 138, in publish
  Module ZPublisher.mapply, line 77, in mapply
  Module ZPublisher.Publish, line 48, in call_object
  Module zope.formlib.form, line 868, in __call__
  Module five.formlib.formbase, line 50, in update
  Module zope.formlib.form, line 849, in update
  Module zope.formlib.form, line 640, in success
  Module plone.app.users.browser.register, line 621, in action_join
  Module plone.app.users.browser.register, line 476, in handle_join_success
AttributeError: validate_email

## Expected behaviour

New user registration should be enabled

## Versions

- OS: Linux Ubuntu Server 18.04

- senaite.core: 1.3.2
- senaite.impress: 1.2.2
- senaite.core.listing: 1.3.0
- senaite.core.supermodel: 1.2.1

## Traceback
```
Traceback (innermost last):
  Module ZPublisher.Publish, line 138, in publish
  Module ZPublisher.mapply, line 77, in mapply
  Module ZPublisher.Publish, line 48, in call_object
  Module zope.formlib.form, line 868, in __call__
  Module five.formlib.formbase, line 50, in update
  Module zope.formlib.form, line 849, in update
  Module zope.formlib.form, line 640, in success
  Module plone.app.users.browser.register, line 621, in action_join
  Module plone.app.users.browser.register, line 476, in handle_join_success
AttributeError: validate_email
```

## Screenshot
![image](https://user-images.githubusercontent.com/56566458/68004947-fe6efa00-fca5-11e9-9519-9a64227edbb8.png)

**Comment by gonzhd** [comment]
I got exactly the same error message

**Comment by ramonski** [comment]
This issue was fixed in https://github.com/senaite/senaite.core/pull/1469 and will be shipped with the next release. At the moment you can only use the source distribution to bypass this issue.

**Comment by xispa** [comment]
Shall we close this one?

**Comment by noob-git** [comment]
> This issue was fixed in #1469 and will be shipped with the next release. At the moment you can only use the source distribution to bypass this issue.

Thank you. I'll be waiting for next release.



> Shall we close this one?

Sure.
